_includes/footer.html

```diff
@@ -9,7 +9,4 @@ <h2 class="newsletters-title">Mantenha-se atualizado!</h2>
       </form>
     </section>
   </div>
-</footer>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/zepto/1.1.6/zepto.min.js"></script>
```

Comment:
I removed 3 scripts that are duplicated in _layouts/default.html

File: https://github.com/free-time/free-time.github.io/blob/master/_layouts/default.html